ansible_collections/arista/avd/roles/eos_designs/templates/base/base.py

```diff
@@ -0,0 +1,559 @@
+                no_password = get(local_user, 'no_password')
+                ssh_key = get(local_user, 'ssh_key')
+                if role is not None:
+                    local_users_dict[name].update({'role': role})
+                if sha512_password is not None:
+                    local_users_dict[name].update({'sha512_password': sha512_password})
+                elif no_password is not None:
+                    local_users_dict[name].update({'no_password': no_password})
+                if ssh_key is not None:
+                    local_users_dict[name].update({'ssh_key': ssh_key})
+            return local_users_dict
+        return None
+
+    @cached_property
+    def clock(self):
+        timezone = get(self._hostvars, 'timezone')
+        if timezone is not None:
+            return {'timezone': timezone}
+        return None
+
+    @cached_property
+    def vrfs(self):
+        return {self._mgmt_interface_vrf: {'ip_routing': get(self._hostvars, 'mgmt_vrf_routing')}}
+
+    @cached_property
+    def management_interfaces(self):
+        mgmt_interface = get(self._switch_data, 'mgmt_interface')
+        if (
+                mgmt_interface is not None and
+                self._mgmt_ip is not None and
+                self._mgmt_interface_vrf is not None and
+                self._mgmt_gateway is not None
+        ):
+            return {
+                mgmt_interface: {
+                    'description': 'oob_management',
+                    'shutdown': False, 'vrf': self._mgmt_interface_vrf,
+                    'ip_address': self._mgmt_ip, 'gateway': self._mgmt_gateway,
+                    'type': 'oob'
+                }
+            }
+        return None
+
+    @cached_property
+    def tcam_profile(self):
+        tcam_profile = get(self._platform_settings, 'tcam_profile')
+        if tcam_profile is not None:
+            return {'system': tcam_profile}
+        return None
+
+    @cached_property
+    def platform(self):
+        lag_hardware_only = get(self._platform_settings, 'lag_hardware_only')
+        platform = {}
+        if lag_hardware_only is not None:
+            platform.update({'sand': {'lag': {'hardware_only': lag_hardware_only}}})
+        if (
+                get(self._platform_settings, 'trident_forwarding_table_partition') is not None and
+                get(self._switch_data, 'evpn_multicast') is True
+        ):
+            platform.update({'trident': {'forwarding_table_partition':
+                            get(self._platform_settings, 'trident_forwarding_table_partition')}})
+        if platform:
+            return platform
+        return None
+
+    @cached_property
+    def mac_address_table(self):
+        mac_address_table = get(self._hostvars, 'mac_address_table')
+        if mac_address_table is not None:
+            aging_time = get(mac_address_table, 'aging_time')
+            if aging_time is not None:
+                return {'aging_time': aging_time}
+        return None
```

Comment:
```suggestion
        mac_address_table = get(self._hostvars, 'mac_address_table')
        if mac_address_table is None or (aging_time := get(mac_address_table, 'aging_time')) is None:
            return None
        return {'aging_time': aging_time}
```

Reply:
This is not really making it simpler to read.

Reply:
Instead we should use `get` feature of dot-notation, so you can directly read `mac_address_table.aging_time`